opennms-doc/guide-admin/src/asciidoc/text/minion/offheap.adoc

```diff
@@ -3,35 +3,31 @@
 
 === Using Off-heap Storage for Sink Messages
 
-If a _Minion_ loses connectivity with the broker (i.e. _Kafka_ or _ActiveMQ_), then any received messages (i.e. syslog, flows, SNMP traps) are queued in the JVM heap until connectivity is restored.
-This queue is limited by a fixed (and configurable) number of messages.
+If a _Minion_ loses connectivity with the broker (i.e. _Kafka_ or _ActiveMQ_), then any received messages (i.e. syslog, flows, SNMP traps) are queued until connectivity is restored.
+This queue is limited by a fixed (and configurable) number of messages queued in the JVM heap and can optionally queue
+additional messages by persisting directly to the filesystem avoiding heap memory usage.
 Once the queue is full, additional messages will be dropped.
 
 The off-heap storage feature allows us to extend the storage capacity by queuing messages outside of the JVM heap.
 
-NOTE: The current implementation of the off-heap storage supports storing messages in the system memory outside of the heap.
-      No file-system based storage is currently support.
-
 ==== Configuring Off-heap Storage
 
-Install off-heap feature on _Minion_:
+Configure storage limits:
 
 [source, sh]
 ----
-echo 'opennms-core-ipc-sink-offheap' >> "$MINION_HOME/etc/featuresBoot.d/features.boot"
+echo 'offHeapSize = 1GB
+entriesAllowedOnHeap = 100000
+offHeapFilePath =' > "$MINION_HOME/etc/org.opennms.core.ipc.sink.offheap.cfg"
 ----
```

Comment:
We should probably add a note saying that this file size is per module.

Reply:
Will do.